addons/survey/views/survey_survey_views.xml

```diff
@@ -191,74 +191,163 @@
-                <field name="success_count"/>
+            <kanban class="o_survey_survey_view_kanban" sample="1">
+                <field name="active"/>
+                <field name="certification"/>
+                <field name="color"/>
+                <field name="create_date"/>
+                <field name="scoring_type"/>
+                <field name="session_state"/>
                 <field name="success_ratio"/>
-                <field name='active'/>
                 <templates>
                     <div t-name="kanban-box" 
-                        t-attf-class="oe_kanban_color_#{kanban_getcolor(record.color.raw_value)} oe_kanban_card oe_kanban_global_click o_kanban_card_survey 
-                            #{record.certification.raw_value ? 'o_kanban_card_survey_successed' : ''}">
-                        <div class="o_dropdown_kanban dropdown" t-if="widget.editable">
-
-                            <a role="button" class="dropdown-toggle o-no-caret btn" data-toggle="dropdown" data-display="static" href="#" aria-label="Dropdown menu" title="Dropdown menu">
-                                <span class="fa fa-ellipsis-v"/>
-                            </a>
-                            <div class="dropdown-menu" role="menu">
-                                <a role="menuitem" type="edit" class="dropdown-item">Edit Survey</a>
-                                <a t-if="record.active.raw_value" role="menuitem" type="object" class="dropdown-item" name="action_send_survey">Share</a>
-                                <a t-if="widget.deletable" role="menuitem" type="delete" class="dropdown-item">Delete</a>
-                                <div role="separator" class="dropdown-divider"/>
-                                <div role="separator" class="dropdown-item-text">Color</div>
-                                <ul class="oe_kanban_colorpicker" data-field="color"/>
+                        t-attf-class="oe_kanban_color_#{kanban_getcolor(record.color.raw_value)}
+                                      oe_kanban_card oe_kanban_global_click
+                                      o_survey_kanban_card #{record.certification.raw_value ? 'o_survey_kanban_card_certification' : ''}">
+                        <!-- displayed in ungrouped mode -->
+                        <div class="o_survey_kanban_card_ungrouped row mx-0">
+                            <div class="col-lg-2 col-sm-8 col-12 py-0 my-2 my-lg-0">
+                                <div class="d-flex flex-grow-1 flex-column my-0 my-lg-2">
+                                    <span class="font-weight-bold"><field name="title"/></span>
+                                    <span>
+                                        <field name="user_id" widget="many2one_avatar_user"
+                                            options="{'display_avatar_name': True}"/>
+                                        -
+                                        <t t-esc="moment(record.create_date).format('MMM YYYY')"/>
```

Comment:
Not convinced about hardcoded format using moment. 
Did not find many occurrences in the code.

Will it respect user locale? I suspect it will use the browser one.
Also a bit concerned about timezones and all that (even if it's the month so it should be fine but... well).
 
Can't we use widget with formatting options?

Reply:
Anyway I don't think there is any possibility to respect user locale, as there is no "nice" formatting like what we want. Formatting like short, medium, long, ... that respects the locale are not what we want here. Unless there is something I did not see in the various code + babel + formatting stuff I checked.

Reply:
Nothing works actually. Using field, format is not supported. Same for using manually t-esc + widget. Hey, no time to debug kanban fields, so keeping moment from now on. Fixed the value, as it was always now actually.